projects/packages/search/src/dashboard/components/record-meter/test/notice-box.test.jsx

```diff
@@ -0,0 +1,101 @@
+/**
+ * @jest-environment jsdom
+ */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+/**
+ * Internal dependencies
+ */
+import { NoticeBox } from 'components/record-meter/notice-box';
+
+describe( 'with notices to display', () => {
+	test( 'not-indexed notice is displayed', () => {
+		render(
+			<NoticeBox
+				recordCount={ 20 }
+				planRecordLimit={ 100 }
+				hasBeenIndexed={ false }
+				hasValidData={ true }
+				hasItems={ true }
+			></NoticeBox>
+		);
+		expect( screen.getByText( 'not yet been indexed' ) ).toBeVisible();
```

Comment:
I think we'll have to use regular expressions for these, because `getByText` matches the whole content of the element. Try something like:

`expect( screen.getByText( /not yet been indexed/i )`

(the `i` is for case-insensitive)